.circleci/config.yml

```diff
@@ -152,7 +152,7 @@ jobs:
   integration_test:
     working_directory: ~/build
     environment:
-      TESTRUNNER_DOCKER_IMAGE: 104059736540.dkr.ecr.us-west-2.amazonaws.com/mbed/sdk-testrunner:master
+      TESTRUNNER_DOCKER_IMAGE: 104059736540.dkr.ecr.us-west-2.amazonaws.com/mbed/sdk-testrunner
```

Comment:
This isn't used anywhere that I can see

Reply:
Its used in run.sh https://github.com/ARMmbed/mbed-cloud-sdk-javascript/blob/master/test/integration/run.sh#L36